modules/gltf/gltf_document.cpp

```diff
@@ -450,7 +450,7 @@ Error GLTFDocument::_serialize_nodes(Ref<GLTFState> state) {
 }
 
 String GLTFDocument::_sanitize_scene_name(const String &name) {
-	RegEx regex("([^a-zA-Z0-9_ -]+)");
+	RegEx regex("([^a-zA-Z0-9_ -=!%^&*(){}\\[\\]|<>,~]+)");
```

Comment:
Trying to find cases where this breaks. How does it behave with non English node names?

Reply:
Sorry, replied in the top level comment - it'll still strip any non-ASCII characters.